src/community/schemaless-features/mongodb-schemaless/src/test/java/org/geoserver/schemalessfeatures/mongodb/WMSSchemalessMongoTest.java

```diff
@@ -226,7 +226,7 @@ public void testStationsWmsGetFeatureInfo() throws Exception {
                                 "wms?SERVICE=WMS&VERSION=1.1.1"
                                         + "&REQUEST=GetFeatureInfo&FORMAT=image/png&TRANSPARENT=true&QUERY_LAYERS=gs:"
                                         + StationsTestSetup.COLLECTION_NAME
-                                        + "&STYLES&LAYERS=gs:"
+                                        + "&STYLES=point&LAYERS=gs:"
```

Comment:
For this GetFeatureInfo request previous default style did not display features from request.